cns/api.go

```diff
@@ -356,6 +356,7 @@ type NmAgentSupportedApisResponse struct {
 type HomeAzResponse struct {
 	IsSupported bool `json:"isSupported"`
 	HomeAz      uint `json:"homeAz"`
+	APIVersion  uint `json:"apiVersion"`
```

Comment:
This should indicate whether or not we should Enable AZR (per earlier discussion). We can set this from doing:

```go
resp, err := nmaClient.HomeAz(ctx)
if err != nil {
       return errors.Wrap(err, "getting home az")
}
return cns.HomeAzResponse{
       EnableAZR: resp.ShouldEnableAZR(),
}
```

Then this can be directly consumed in clients without further interpretation about the meaning of codes.

Reply:
addressed